src/View/Helper/SchemaHelper.php

```diff
@@ -100,6 +100,28 @@ protected function customControlOptions($name) : array
                     ],
                 ];
 
+            case 'username':
```

Comment:
`username` not necessary an `email`, I would not represent it as email in general

Reply:
should not also be `readonly` in general